ceph.spec.in

```diff
@@ -503,7 +503,7 @@ Requires:       python3-scipy
 Requires:       python2-scipy
 %endif
 %endif
-%if 0%{?rhel} <= 7
+%if 0%{?rhel} && 0%{?rhel} <= 7
```

Comment:
how about just put 
```
%if 0%{?rhel} && 0%{?rhel} == 7
```

as we don't build on RHEL6 or less. and this approach is also consistent with other places in this file.